examples/qec/onequbit_qec.py

```diff
@@ -0,0 +1,28 @@
+"""
+This is the base class for code for one physical qubit
+"""
+
+import cirq
+
+
+class OneQubitCode:
```

Comment:
I feel like this is abstraction is not important for an example, can you just merge it into the `OneQubitShorsCode`?

Reply:
I think it's because we plan on adding more codes besides Shor's as examples in the future.

Reply:
base class removed